continuous_queryies.md

```diff
@@ -0,0 +1,104 @@
+# Continuous Queries
+
+This document lays out continuous queries and a proposed architecture for how they'll work within an InfluxDB cluster.
+
+## Definition of Continuous Queries
+
+Continuous queries serve two purposes in InfluxDB: denormalizing data into many series or into a single series and aggregating and downsampling data. We refer to these as `fan-out`, `fan-in`, or `aggregation`. Before we get to the continuous query examples, we need to define the `INTO` syntax of queries.
```

Comment:
Can you split the sentence after the colon into a list? There's a lot of "ors" and "ands". e.g.

1. denormalizing data into many series or into a single series
2. aggregating and downsampling data

Then the next sentence lists 3 ways we refer to it but we only mentioned 2 things before.